<ui version="4.0" >
 <class>weatherConfig</class>
 <widget class="QWidget" name="weatherConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Weather Applet Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Select data source:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="pluginComboList" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Type in a location:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="locationEdit" >
         <property name="acceptDrops" >
          <bool>false</bool>
         </property>
         <property name="frame" >
          <bool>true</bool>
         </property>
         <property name="urlDropsEnabled" >
          <bool>false</bool>
         </property>
         <property name="trapEnterKeyEvent" stdset="0" >
          <bool>true</bool>
         </property>
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="validateButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&amp;Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Selected places:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="weathersourceTree" >
       <property name="windowModality" >
        <enum>Qt::NonModal</enum>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors" >
        <bool>false</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="autoExpandDelay" >
        <number>-1</number>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights" >
        <bool>false</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="animated" >
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Remove Place</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Update weather information every</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="weatherUpdateSpin" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix" >
        <string> minutes</string>
       </property>
       <property name="prefix" >
        <string/>
       </property>
       <property name="minimum" >
        <number>30</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="singleStep" >
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Select wind format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="windOptionsComboList" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>validateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
